### ISSUE TITLE
Group Tabs by Domain

### ISSUE THREAD

**Issue by bkonia** [mbnuqw/sidebery#56]
Great extension!

Could you please add an option to automatically group tabs by domain?

**Comment by mbnuqw** [comment]
Yes, this will be possible after next (v3.0.0) release. Also, I plan to add include/exclude rules for group pages to auto arrange opened tabs.

**Comment by bkonia** [comment]
That's great news, thanks!

**Comment by h5vx** [comment]
Is it already working? Cannot find that option...

**Comment by Dorieto** [comment]
Has this option been added? I don't see an option for it anywhere

**Comment by mbnuqw** [comment]
This option hasn't been added yet, but it's planned, albeit with a low priority.

**Comment by ericpa06** [comment]
I see there is already an issue on that, and indeed, it would be really cool feature, especially because there aren't many vertical tab extensions that do that. One extension I could find that does that is [Sidebar+](https://addons.mozilla.org/en-US/firefox/addon/sidebar_plus/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search), although their implementation on this particular feature is okay, the rest of the extension isn't that good at all, it is nowhere near extensions like Sidebery or Tree Style Tab.

![image](https://github.com/mbnuqw/sidebery/assets/6937720/78ae3ca6-113c-4b2d-a8df-358ef046911c)

So adding this feature would make this extension just perfect, honestly I hope it gets implemented soon :)

**Comment by mustafa89** [comment]
This would be great to have.

**Comment by DudeDude2** [comment]
Is there any update for this? It would be a really nice feature.

**Comment by immagiov4** [comment]
This would be really useful indeed

**Comment by lightningHoneypot** [comment]
it's been 7 years already, when are we getting it?

**Comment by animaone** [comment]
select all tabs and sort by url

**Comment by nimo23** [comment]
I am not sure wheter it is related but maybe we can use https://github.com/nitzanpap/auto-tab-groups after https://github.com/mbnuqw/sidebery/pull/2234 was merged. 

Another way:
Sideberry can already group tabs—though it merely marks them **with colors** rather than moving similar tabs into a new group (or a parent tab). Perhaps this grouping function could be used for that purpose. 

And even another way:
Let Sideberry use the local WebLLM tool to manage the grouping..
